tests/acceptance/acceptancesuite.js

```diff
@@ -63,6 +63,31 @@ test('navigating and refreshing mantains that page number', async t => {
   await t.expect(pageNum).eql('Page 2');
 });
 
+const spellCheckLogger = RequestLogger({
+  url: /v2\/accounts\/me\/answers\/vertical\/query/
+});
+test.requestHooks(spellCheckLogger)('spell check flow', async t => {
+  const searchComponent = VerticalPage.getSearchComponent();
+  await searchComponent.enterQuery('varginia');
+  await searchComponent.submitQuery();
+
+  // Check that clicking spell check resets pagination to page 1
```

Comment:
What do you think about moving this comment down to the next block of code? (above line 80)

Reply:
I kind of like it up here, since the only reason I'm changing the page from 1 to 2 is to test this, but don't really have strong feelings

Reply:
ah, got it